app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/BonusSectionView.kt

```diff
@@ -119,12 +119,16 @@ private fun BonusBannerText(
   annotatedString: AnnotatedString,
   inlineContent: Map<String, InlineTextContent>,
 ) {
-  Column(modifier = Modifier
-    .padding(start = 16.dp, top = 44.dp)
-    .width(240.dp)
+  Column(
+    modifier = Modifier
+      .padding(start = 16.dp, top = 44.dp)
+      .width(240.dp)
   ) {
     AptoideOutlinedText(
-      text = stringResource(id = R.string.bonus_banner_title, "20"), //TODO Hardcoded value (should come from backend in the future)
+      text = stringResource(
+        id = R.string.bonus_banner_title,
+        "20"
+      ), //TODO Hardcoded value (should come from backend in the future)
```

Comment:
Move the comment on the previous line next to `"20"`

Reply:
Fixed